subtests/docker_cli/negativeusage/negativeusage.py

```diff
@@ -120,12 +120,12 @@ def init_subcntrs(self):
             raise xceptions.DockerTestNAError("Failed to initialize %s"
                                               % self.config_section)
         # Throw away cntr, all we need is the CID
-        cntr = mustpass(dockercmd.DockerCmd(self, 'run',
-                                            ['--detach',
-                                             self.sub_stuff['FQIN'],
-                                             'true']).execute())
+        result = mustpass(dockercmd.DockerCmd(self, 'run',
+                                              ['--detach',
+                                               self.sub_stuff['FQIN'],
+                                               'true']).execute())
         # Only the CID is needed
-        self.sub_stuff['STPCNTR'] = cntr.stdout.splitlines()[-1].strip()
```

Comment:
R0204:123,8: Base.init_subcntrs: Redefinition of cntr type from dockertest.dockercmd.AsyncDockerCmd to autotest.client.shared.utils.CmdResult